mimesis/providers/structured.py

```diff
@@ -113,7 +109,7 @@ def html_attribute_value(self, tag, attribute):
                 'Attribute type {} is not implemented'.format(value))
         return value
 
-    def json(self, items=5, max_depth=3, recursive=False):
+    def json(self, items: int=5, max_depth: int=3, recursive: bool=False) -> str:
```

Comment:
Too many errors.

You should be aware of what you are writing, otherwise there will be a mess and I will not be able to merge your PR and your work will be done in vain.

Reply:
As i say, i have created type for json. Look at `mimesis.typing`. branch: `type-hinting`.